package.json

```diff
@@ -2,9 +2,6 @@
   "private": true,
   "name": "parent",
   "version": "0.0.0",
-  "dependencies": {
-    "node-pty": "^0.6.10"
-  },
   "devDependencies": {
```

Comment:
why is this now removed ?

Reply:
The rebuild script will write package.json with node-pty automatically by default, so we don't need to duplicate it. Additional modules can be passed with `modules` option.

Reply:
OK got it thx, that's nice